Rakefile

```diff
@@ -1,9 +1,20 @@
-require 'cucumber/rake/task'
+# require 'cucumber/rake/task'
 require 'rspec/core/rake_task'
 require 'rubocop/rake_task'
+require 'data_mapper'
 
 RuboCop::RakeTask.new :cop
 RSpec::Core::RakeTask.new :spec
-Cucumber::Rake::Task.new :cuke
+# Cucumber::Rake::Task.new :cuke
 
-task default: [:cop, :spec, :cuke]
\ No newline at end of file
+task default: [:cop, :spec]
+
+task :auto_upgrade do
+  DataMapper.auto_upgrade!
+  puts 'auto_upgrade complete (no data loss)'
+end
+
+task :auto_migrate do
+  DataMapper.auto_migrate!
+  puts 'auto_migrate complete (data could have been lost)'
+end
```

Comment:
rake auto_migrate and rake auto_upgrade do not work
try this:
require_relative 'data_mapper_set_up' instead of require 'data_mapper'
and I'll point that changes you need to make in data_mapper_set_up